test/reporter-html/diff.js

```diff
@@ -0,0 +1,163 @@
+QUnit.module( "diff" );
```

Comment:
Our current diff is only relevant to the HTML reporter as its output is HTML. These tests intend to progress the coverage on the diff functions, but they are meant for what's expected on the reporter, meaning we could change it for any other library in a future.